### PR TITLE
fix(curriculum): fixed broken link and first answer for Odin Links and Images Question H 

### LINKED ISSUE #52670: Fix Broken link in Odin project challenge and update first answer - Links and Images Question H
### Describe the Issue

We recently got a [report](https://forum.freecodecamp.org/t/links-and-images-links-and-images-question-h/656943/2) of a broken link in the [Links and Images Questions H challenge](https://www.freecodecamp.org/learn/the-odin-project/top-links-and-images/links-and-images-question-h).

Under the assignments section, there is a link to read an article about the different image formats.
Screenshot 2023-12-20 at 3 11 59 PM

However, that link no longer works. 
We should use a link from our own fcc publication instead to fix this.

## How to contribute 

This is a small fix so I am opening this up for contribution for first timers only.
Please read through the fix below and read through our [contributing](https://contribute.freecodecamp.org/#/) guide.
There will be no assignment for this issue. 
Feel free to work on it and create a PR. 

## How to fix the issue

Here is the file you will be working with
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/16-the-odin-project/top-links-and-images/links-and-images-question-h.md

Replace the old link here:
```
# --assignment--

Read about the  <a href="https://www.internetingishard.com/html-and-css/links-and-images/#image-formats" target="_blank">four main image formats</a> that can be used on the web.
```

with this new change here

```
# --assignment--

Read about the  <a href="https://www.freecodecamp.org/news/image-file-types-picture-format-extensions-jpeg-gif-png-svg-tiff/" target="_blank">four main image formats</a> that can be used on the web.
```

Also, let's change the first multiple choice answer here

```
## --answers--

TIFF, GIF, PNG, and SVG.

---
```

and change it to this new option here

```
## --answers--

MOV, GIF, PNG, and SVG.

---
```

### Affected Page

https://www.freecodecamp.org/learn/the-odin-project/top-links-and-images/links-and-images-question-h

### Your code

N/A

### Expected behavior

The link should be replaced with this one here

https://www.freecodecamp.org/news/image-file-types-picture-format-extensions-jpeg-gif-png-svg-tiff/

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Snp-Rj-Ind-code-error-420**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ]  I have tested these changes either locally on my machine, or GitPod.

Closes #52670

### CODE REVIEW

**Review by huyenltnguyen — COMMENTED** [review]
@Snp-Rj-Ind-code-error-420 Thank you for the PR.

Please help update the PR title to follow our convention here: https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title.

File: curriculum/challenges/english/16-the-odin-project/top-links-and-images/links-and-images-question-h.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ This is how the The Odin Project logo example you used earlier looks with an alt
 
 # --assignment--
 
-Read about the  <a href="https://www.internetingishard.com/html-and-css/links-and-images/#image-formats" target="_blank">four main image formats</a> that can be used on the web.
+Read about the  <a href="https://www.freecodecamp.org/news/image-file-types-picture-format-extensions-jpeg-gif-png-svg-tiff/" target="_blank">four main image formats</a> that can be used on the web.
```
[the remark this answers is not in the document]
Comment (Snp-Rj-Ind-code-error-420) [reply curriculum/challenges/english/16-the-odin-project/top-links-and-images/links-and-images-question-h.md]: Any more suggestions to be implemented 😁

**Review by huyenltnguyen — APPROVED** [review]

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Comment by gikf** [comment]
Hi @Snp-Rj-Ind-code-error-420. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**